File: hobby-tracker-web-api/hobby-tracker-web-api/Database/Entities/Habit.cs
namespace hobby_tracker_web_api.Database.Entities;

public class Habit
{
  public int Id { get; set; }
  public string Name { get; set; }
  public List<Day> Days { get; set; } = new List<Day>();
  public int Goal { get; set; }
  public int Achieved { get; set; }
}

File: src/2.entities/Habit.ts
import Day from "./Day";

class Habit {
 constructor(id: number, name: string, days: Day[], goal: number, achieved: number) {
  this.id = id;
  this.name = name;
  this.days = days;
  this.goal = goal;
  this.achieved = achieved;
 }

 id: number
 name: string
 days: Day[]
 goal: number
 achieved: number
}

export default Habit;